### ISSUE TITLE
Adding a github channel in the Discord

### ISSUE THREAD

**Issue by fireFerry** [rockyhawk64/CommandPanels#2]
**Is your feature request related to a problem? Please describe.**
No, but it's nice to have a github channel with all the github updates

**Describe the solution you'd like**
Create a github channel, add a github webhook and then new updates to the code will be posted by the webhook in that particular channel

**Describe alternatives you've considered**
Not doing it

**Additional context**

**Comment by TheLonelyWolf1** [comment]
Good Idea...i could do all the discord-related work, but Rocky would need to include the url into the repo's settings
